engine/server/engine/main.go

```diff
@@ -134,8 +134,9 @@ func runMain() error {
 		return stacktrace.Propagate(err, "Failed to create an enclave manager for backend type '%v' and config '%+v'", serverArgs.KurtosisBackendType, backendConfig)
 	}
 
-	// TODO: replace with persistent client so that we can get logs even after enclave is stopped.
+	// TODO: replace with persistent client so that we can get logs even after enclave is stopped
 	logsDatabaseClient := kurtosis_backend.NewKurtosisBackendLogsDatabaseClient(kurtosisBackend)
```

Comment:
What's this? Are we returning vector here? Should we change "database" for "aggregator" then?

Reply:
The `LogsDatabaseClient` is the interface we use to access logs from wherever they are stored. When we had loki database, `LogsDatabaseClient` would be pulling from there. Currently, and until the persistent volume is implemented the `LogsDatabaseClient` pulls from `docker logs` (hence KurtosisBackendLogsDatabase..) When the persistent volume is implemented, `LogsDatabaseClient` will pull from that impl